### PR TITLE
Add Form

### PR DESCRIPTION
**Pull request by aron-net**
These are the proposed changes :recycle: for this project :arrow_down :

Created a method called current_user in the ApplicationController to make current user data available to all controllers. :1234:
Generated controllers to handle form actions for comments and likes
Created forms to perform the following functions :heavy_check_mark: 
Creates a Post on behalf of the current_user :bust_in_silhouette: .
Creates a comment on behalf of the current_user :right_anger_bubble: :bust_in_silhouette: .
Allow Users to add likes to Posts. :heart_decoration:

### CODE REVIEW

File: app/views/posts/show.html.erb [hunk 1]
```diff
@@ -4,11 +4,16 @@
...
         <h5><%= comment.user.name %>: <%= comment.text %> </h5>
     <% end %>
+   <div class="add-comment">
+      <h2>Add new comment</h2>
+      <%= render "comments/comment" %>
+    </div>
 </div>
-<a href='/users/'><button>Go back</button></a>
\ No newline at end of file
+<%# <a href='/users/'><button>Go back</button></a> %>
```
Comment (rustam-tolipov) [inline app/views/posts/show.html.erb]: Your project is working perfect. However, there are some changes need to be done. 

- [ ] Kindly remove commented codes from your project. :)

File: config/routes.rb [hunk 2]
```diff
@@ -1,13 +1,19 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do 
+  # get 'likes/create'
+  # get 'comments/new'
+  # get 'comments/create'
```
Comment (rustam-tolipov) [inline config/routes.rb]: - [ ] Kindly remove commented codes from your project. :)

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @aron-helu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 2 above)
Comment (aron-net) [reply config/routes.rb]: Already implemented it! Thanks for the feedback! :smile:

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @aron-helu 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**